code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/monitor-fix-issues-in-scan.adoc

```diff
@@ -0,0 +1,373 @@
++
+3. *Additional Information*: Displays columns of the information regarding the issue.
++
+* *Secret type*: Displays the severity level of the exposed secret in the code.
++
+* *Risk Factors*: For Secrets there are three types of risk factors.
++
+** IsPublic: Identifies if the repository storing the secret is publicly accessible or private.
++
+** LastModifiedBy: The name of last contributing user before identifying the issue.
++
+** ModifiedOn: The last modification date of the relevant code.
++
+* *First Detected*: The timestamp of the issue when found.
+
+
+* *Licensing Resource Block*
++
+As a licensing issue, there is an extensive information in the resource block for a packages using the open source licensing.
++
+image::proj-5.png[width=800]
++
+1. *Package Name and Path*: Displays the package name and it's code path.
++
+2. *Total number of Issues*: Displays the total number of issues identified in the package.
++
+3. *Additional Information*: Displays columns of the information regarding the issue.
++
+* *Repository*: See the repository path.
++
+* *Policy*: Displays severity of the policy violation when using an open source licensing package.
++
+* *License Type*: Displays the origin of license if it is originating from the root package or the a dependent package.
++
+* *Package*: The name of package.
++
+* *First Detected*: The timestamp of the issue when found.
+
+
+* *Build Integrity Resource Block*
++
+As a Build Integrity issue, there is an extensive information in the resource block.
++
+image::proj-6.png[width=800]
++
+1. *Branch Name and Path*: Displays the branch name and it's code path.
++
+2. *Total number of Issues*: Displays the total number of issues identified in the repository.
++
+3. *Additional Information*: Displays columns of the information regarding the issue.
++
+* *Policy*: Displays the severity level of non-conformant policy in the code.
++
+* *First Detected*: The timestamp of the issue when found.
+
+=== Resource Explorer and Fix Cart
+
+In helping you make educated decision, Prisma Cloud provides you with granular information on each issue within Resource Explorer. Later each of the issues are remediated on Fix Cart.
+
+==== Resource Explorer
+
+The information on Resource Explorer enables you to make an educated decision on the security violation and understand if the violation has any connection as a dependency on other resources within the repository while exploring the change log of the resource.
+You can view this contextualized information across four tabs.
+
+* *Details*: Helps you understand the connection between resources while enabling you to make informed decisions if the connection is at risk or if it is necessary.
++
+image::proj-7.png[width=400]
+
+* *Errors*: Enables you to review security violations with the package severity threshold and utilize the information to either suppress or prioritize it.
```

Comment:
now it's Issues and it shows all statuses and not only Errors across all resource types